README.rst

```diff
@@ -1208,37 +1206,27 @@ As as side effect, the tool will validate the files for format errors, also some
 ordering of the types of the steps.
 
 
-.. _Migration from 3.x.x:
+.. _Migration from 4.x.x:
```

Comment:
I would leave the old migration section, not everybody updates to each version of the library at every major release